.github/actions/gui-test/pyi/action.yml

```diff
@@ -8,7 +8,7 @@ runs:
       run: pipenv run pip install mypy black isort
     - name: Generate pyi
       shell: bash
-      run: cp tools/gui/generate_pyi.py pyi_temp.py && pipenv run python pyi_temp.py && rm pyi_temp.py
```

Comment:
why was it through a copy @dinhlongviolin1 ?

Reply:
Because of the import of taipy itself.